.github/workflows/development.yaml

```
@@ -4,10 +4,8 @@
 
  jobs:
   build:
-    runs-on: ubuntu-18.04
-    strategy:
-      matrix:
-        python: [3.8]
+    runs-on: [self-hosted, linux]
```

```suggestion
    runs-on: ubuntu-20.04
```
We cannot use self hosted runners in public repos